rust/azure_iot_operations_services/tests/state_store_client_test.rs

```diff
@@ -408,6 +408,7 @@ async fn state_store_set_conditions_network_tests() {
                     SetOptions {
                         expires: Some(Duration::from_secs(10)),
                         set_condition: SetCondition::OnlyIfDoesNotExist,
+                        ..Default::default()
```

Comment:
would there be any test we can add for the new field? I don't think so, but figured I'd check